<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientControlPage</class>
 <widget class="QWidget" name="ClientControlPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Address Book</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="Upd8Control">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Update Settings</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>19</y>
          <width>251</width>
          <height>301</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>120</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>110</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="title">
            <string>Auto Update</string>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget_5">
            <property name="geometry">
             <rect>
              <x>7</x>
              <y>19</y>
              <width>231</width>
              <height>51</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QCheckBox" name="checkBoxupd8">
               <property name="text">
                <string>Check for Updates on Startup</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="AU_apply">
               <property name="text">
                <string>Apply</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QPlainTextEdit" name="dDIR">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>80</y>
              <width>161</width>
              <height>31</height>
             </rect>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>84</y>
              <width>71</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Data</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>150</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="title">
            <string>Manual Update</string>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget_4">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>19</y>
              <width>231</width>
              <height>151</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QPushButton" name="chck4_upd8">
               <property name="text">
                <string>Check for Update</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="dwngrd_opt">
               <property name="text">
                <string>Downgrade Current Client</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>85</height>
                </size>
               </property>
               <property name="title">
                <string>Version</string>
               </property>
               <widget class="QWidget" name="horizontalLayoutWidget_2">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>40</y>
                  <width>211</width>
                  <height>31</height>
                 </rect>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QPlainTextEdit" name="instVER">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>91</width>
                     <height>25</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPlainTextEdit" name="liveVER">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>91</width>
                     <height>25</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QLabel" name="label">
                <property name="geometry">
                 <rect>
                  <x>40</x>
                  <y>20</y>
                  <width>61</width>
                  <height>21</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Installed</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_2">
                <property name="geometry">
                 <rect>
                  <x>150</x>
                  <y>20</y>
                  <width>61</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Latest</string>
                </property>
               </widget>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="ClieControl">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Interface Options</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>19</y>
          <width>251</width>
          <height>301</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="title">
            <string>select</string>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget_8">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>19</y>
              <width>231</width>
              <height>271</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QRadioButton" name="minCLIE">
                 <property name="text">
                  <string>Minimal</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="plusCLIE">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Plus</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="liteCLIE">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Lite</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="CS_submit">
               <property name="text">
                <string>Submit</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Preview</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="cliePREV">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>144</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="TestControl">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Testing/Bug</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>19</y>
          <width>251</width>
          <height>301</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="title">
            <string>Pre-Release Test </string>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget_7">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>231</width>
              <height>121</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 0, 0);</string>
               </property>
               <property name="text">
                <string>(For Developer)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="optin_test">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Pre-Release and Test versions</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="plainTextEdit_2">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>75</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>false</bool>
               </property>
               <property name="plainText">
                <string>Update preRelease/Test version available</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="title">
            <string>Bug-Reporting</string>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget_6">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>231</width>
              <height>115</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Input Issues</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="br_input">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>65</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>65</height>
                </size>
               </property>
               <property name="plainText">
                <string>Please send your error to contact@luxcoin.tech</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="BR_submit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Submit</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
